recipes/motifcluster/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "motifcluster" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b63509e0f970a5fc67d41ec77e17b1b49cdd0576491754f2fcc91dac5f01ba39
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - networkx >=2.4
+    - numpy >=1.18.3
+    - python >=3.5
+    - scikit-learn >=0.22.2
+    - scipy >=1.4.1
+
+test:
+  imports:
+    - motifcluster
+
+about:
+  home: "https://github.com/WGUNDERWOOD/motifcluster"
+  license: GPLv3
```

Comment:
[SPDX](https://spdx.org/licenses/) wants either `GPL-3.0-only` or `GPL-3.0-or-later` depending on which version range is being allowed.

Reply:
Fixed: changed to `GPL-3.0-only` in commit 4e0b02